### PR TITLE
Intercom CartoDB/cartodb-central/issues/1377

### PR DESCRIPTION
**Pull request by juanignaciosl**
This adds Intercom integration at dashboard.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
@gfiorav CR this, please.

File: app/views/admin/visualizations/index.html.erb [hunk 1]
```diff
@@ -34,4 +34,6 @@
   <div class="SupportBanner" id="support-banner"></div>
 <% end %>
 
+<%= render 'admin/shared/intercom' %>
```
Comment (gfiorav) [inline app/views/admin/visualizations/index.html.erb]: How about rendering the partial only if the conditions are met?

**Comment by gfiorav** [comment]
📦 ready! Made a minor organisational comment

File: app/views/admin/visualizations/index.html.erb (hunk 1 above)
Comment (juanignaciosl) [inline app/views/admin/visualizations/index.html.erb]: If integration is needed at another page condition would be duplicated.

Comment (gfiorav) [inline app/views/admin/visualizations/index.html.erb]: True. Maybe I'd use a helper to encapsulate al the check logic in one method, and still place the check outside. Something about having a partial renderer that might not render when you call it puts me off. 

Super minor! Ignore if you please :)

**Pull request merged**